controller/controller.go

```diff
@@ -378,7 +379,7 @@ func ScaleDown(logger logrus.FieldLogger, mr *models.MixedMetricsReporter, db pg
 	var deletionErr error
 
 	for _, roomName := range idleRooms {
-		err := deletePod(logger, mr, clientset, redisClient, scheduler.Name, roomName)
+		err := deletePod(logger, mr, clientset, redisClient, scheduler.Name, scheduler.Game, roomName, "scale_down")
```

Comment:
It is better if the reasons are at models/contants.go and you call the variables